src/models/ApplicationData.ts

```diff
@@ -7,4 +7,5 @@ export const APPLICATION_DATA_KEY = 'appeals';
 export interface ApplicationData {
     appeal: Appeal;
     navigation: Navigation;
+    submittedAppeal?: Appeal;
```

Comment:
Why store another appeal? Couldn't we just use a flag that returns true as suggested by the spike?

Reply:
The flag in itself would mean that we would have to somehow retrieve the appeal from the api. I took the best of both solutions described in the spike and implemented this instead. No required api calls and simple implementation

Reply:
Also, we are not storing another appeal, we are simply moving the submitted appeal to a 'confirmed' stage while we clear the 'build' stage for the new appeal